tests/sqllogictests/suites/base/05_ddl/05_0003_ddl_alter_table.test

```diff
@@ -130,7 +130,7 @@ INSERT INTO TABLE `05_0003_at_t4` values('a', 'b', ['c1', 'c2'], ('d1', 'd2'))
 query TT
 SHOW CREATE TABLE `05_0003_at_t4`
 ----
-05_0003_at_t4 CREATE TABLE "05_0003_at_t4" ( a VARCHAR NOT NULL, b VARCHAR NULL, c ARRAY(STRING) NULL, d TUPLE(1 STRING, 2 STRING) NULL ) ENGINE=FUSE
+05_0003_at_t4 CREATE TABLE "05_0003_at_t4" (   a VARCHAR NOT NULL,   b VARCHAR NULL,   c ARRAY(VARCHAR NULL) NULL,   d TUPLE(1 VARCHAR NULL, 2 VARCHAR NULL) NULL ) ENGINE=FUSE
```

Comment:
seems there's an extra space?

Reply:
sqllogictests will ignore extra space. uh..  It's so weird.